database/database.go

```diff
@@ -241,16 +248,19 @@ func (t *DBTx) GetTaskBehaviors(id string, forUpdate bool) ([]model.AsyncTaskBeh
 	return behaviors, nil
 }
 
+var baseTaskStatusSelect squirrel.SelectBuilder = psql.Select(
+	"status", "detail", "created_date at time zone (select current_setting('TIMEZONE'))",
+).From("async_task_status")
+
 // GetTaskStatuses fetches a tasks's list of statuses from the DB by ID, ordered by creation date
 func (t *DBTx) GetTaskStatuses(id string, forUpdate bool) ([]model.AsyncTaskStatus, error) {
-	query := `SELECT status, detail, created_date at time zone (select current_setting('TIMEZONE')) AS created_date
```

Comment:
So, it wasn't actually ever required is what I figured out, because the `AT TIME ZONE` isn't actually an aggregate, so postgresql won't change the name of the column.